shoes-core/lib/shoes/packager.rb

```diff
@@ -25,6 +26,17 @@ def run(path)
       @backend.run(path)
     end
 
+    def configure_gems
+      @backend.gems = []
+      return unless defined?(::Bundler)
+
+      @backend.gems = ::Bundler.environment.specs.map(&:name) - ["shoes"]
```

Comment:
might we need all our gems here or just shoes? Not sure what the direct problem is